app/src/src/screens/createItem/CreateCommunityScreen.jsx

```diff
@@ -64,7 +58,7 @@ class CreateCommunityScreen extends React.Component {
     const basicData = {
       title: this.state.title,
       content: this.state.body,
-      community: this.state.user.community,
+      community: this.state.community,
```

Comment:
I noticed that it was saving the user's community and not the community "Posting in" so I changed it. I can undo the change if it's supposed to save the user's community instead